tests/cypress/integration/canvas3d_functionality/case_63_canvas3d_functionality_control_button_mouse_interaction.js

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName } from '../../support/const_canvas3d';
+
+context('Canvas 3D functionality. Control button. Mouse interaction.', () => {
+    const caseId = '63';
+    const screenshotsPath =
+        'cypress/screenshots/canvas3d_functionality/case_63_canvas3d_functionality_control_button_mouse_interaction.js';
+
+    function compareImages(imgBefore, imgAfter) {
+        cy.compareImages(`${screenshotsPath}/${imgBefore}`, `${screenshotsPath}/${imgAfter}`).then((diffPercent) => {
+            expect(diffPercent).to.be.gt(0);
+        });
+    }
```

Comment:
We already have such the function in ``tests/cypress/integration/canvas3d_functionality/case_56_canvas3d_functionality_basic_actions.js``.

Wouldn't it better to import it from there?

Reply:
I moved this function to command file due to running a ```it()``` if import it from the ```case_56```.